controller.rb

```diff
@@ -0,0 +1,41 @@
+get '/' do
```

Comment:
consider moving these routes into separate controllers. What you have works! Just makes it a little easier to keep organized as your app grows in complexity.